src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,32 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+}) => {
+  const {
+    title,
+    user,
+    completed,
+  } = todo;
+
+  return (
+    <article className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': completed,
+      },
+    )}
```

Comment:
Would be better to create varibale for thsi classname

```suggestion
  const todoClassName = classNames('TodoInfo', {
    'TodoInfo--completed': completed,
  });
 
  return (
    <article className={todoClassName}
```